backend/clubs/views.py

```diff
@@ -2629,14 +2635,50 @@ def create_tickets(self, request, *args, **kwargs):
         # Ticket prices must be non-negative
         if any(item.get("price", 0) < 0 for item in quantities):
             return Response(
-                {"detail": "Ticket price cannot be negative."},
+                {"detail": "Ticket price cannot be negative"},
+                status=status.HTTP_400_BAD_REQUEST,
+            )
+
+        # Group discounts must be between 0 and 1
+        if any(
+            item.get("group_discount", 0) < 0 or item.get("group_discount", 0) > 1
+            for item in quantities
+        ):
+            return Response(
+                {"detail": "Group discount must be between 0 and 1"},
+                status=status.HTTP_400_BAD_REQUEST,
+            )
+
+        # Min group sizes must be greater than 1
+        if any(item.get("group_size", 2) < 1 for item in quantities):
+            return Response(
+                {"detail": "Min group size must be greater than 1"},
                 status=status.HTTP_400_BAD_REQUEST,
             )
 
```

Comment:
I'm enforcing that groups must be larger than 1. Do we agree with this?

Reply:
Did you mean <= 1 here?

Reply:
Yup, thanks for catching.